tests/testthat/test-formatters.R

```diff
@@ -1,13 +1,7 @@
-everything <- 42
-g <- function() {
-  log_info("Hi {everything}")
-}
-f <- function() {
-  log_info("Hi %s", everything)
-}
 
 test_that("glue works", {
   local_test_logger(formatter = formatter_glue)
+  a <- 43
```

Comment:
so what's the meaning of 43? :sweat_smile:

Reply:
Haha, I just changed it because I think it's good to use a different value than 42 to hint that we're testing a different mechanism. And while it doesn't really apply here, there's always some risk to reusing the same test value too many times, because it won't catch bugs where the previous value bleeds through to the next expectation.